libs/build_api_v2.py

```diff
@@ -99,31 +99,45 @@ def build_region_summary(
     )
 
 
+ACTUALS_NAME_TO_COMMON_FIELD = {
+    "cases": CommonFields.CASES,
+    "deaths": CommonFields.DEATHS,
+    "positiveTests": CommonFields.POSITIVE_TESTS,
+    "negativeTests": CommonFields.NEGATIVE_TESTS,
+    "contactTracers": CommonFields.CONTACT_TRACERS_COUNT,
+    "hospitalBeds": CommonFields.HOSPITAL_BEDS_IN_USE_ANY,
+    "icuBeds": CommonFields.ICU_BEDS,
+    "newCases": CommonFields.NEW_CASES,
+    "vaccinesDistributed": CommonFields.VACCINES_DISTRIBUTED,
+    "vaccinationsInitiated": CommonFields.VACCINATIONS_INITIATED,
+    "vaccinationsCompleted": CommonFields.VACCINATIONS_COMPLETED,
+}
+
+
+METRICS_NAME_TO_COMMON_FIELD = {
+    "contactTracerCapacityRatio": CommonFields.CONTACT_TRACERS_COUNT,
+    "caseDensity": CommonFields.CASES,  # TODO(chris): Fix this
+    "infectionRate": CommonFields.CASES,  # TODO(chris): Fix this
+    "testPositivityRatio": CommonFields.TEST_POSITIVITY,
+    "icuHeadroomRatio": CommonFields.CASES,  # TODO(chris): Fix this
```

Comment:
Should probably be `CommonFields.CURRENT_ICU_TOTAL`

Reply:
Fixed in f4ef93a